front-end/src/app/shared/models/schb-transaction.model.ts

```diff
@@ -52,7 +52,12 @@ export class SchBTransaction extends Transaction {
   override apiEndpoint = '/transactions/schedule-b';
 
   override getFieldsNotToValidate(): string[] {
-    return ['back_reference_tran_id_number', 'back_reference_sched_name', ...super.getFieldsNotToValidate()];
+    return [
+      'beneficiary_committee_name',
```

Comment:
Is this a required field that the user doesn't provide?

Reply:
Yes, it is similar to the donor_committee_fec_id and donor_committee_name. The plan was to populate the name when the fec file is created.